README.md

```diff
@@ -1,23 +1,118 @@
 # Kuri EDU
-A minimal kuri robot that exposes kuri's subsystems without any autonomous behavior
+This repository contains software and activities intended to be used for
+educational purposes.  The kuri_edu folder contains a ROS package that provides 
+a minimal running robot framework and the activities folder contains ideas and
+instructions for extending the robot behavior
 
-## Launch Files
- * kuri_remote.launch: Launches a Kuri that can be driven with the standard keyboard_teleop node
- * kuri_edu.launch: Launches kuri_remote along with the following subsystems
-   * madmux: Kuri's camera
+## Stand-alone launch Files
+ * kuri_edu.launch: Launches Kuri, and the nodes required to operate Kuri with 
+minimal autonomous behavior.  This launch file glues together several other launch files:
+    * kuri_drive.launch - Runs the hardware abstraction layer and depth-sensor.
+    * madmux_daemon.launch - Provides a hardware abstraction layer to Kuri's camera
+    * vision_bridge.launch - Performs face and object detection
+
+## Getting Started:
+Before attempting any activities, you should make sure you're able to launch the
+kuri EDU software without any modifications.
+
+Start by establishing an SSH connection to Kuri
+`ssh mayfield@kuri-SERIAL_NUMBER`
```

Comment:
* This is not accurate. The serial number is something like `KR118160000339`. You probably wanna tell them to get the hostname from the mobile app or to construct is as `kuri-LAST_7_DIGITS_OF_SN` (assuming DVT+ robot)
* Connecting to Kuri is gonna be a whole topic in the main `documentation`, so maybe link to that eventually.